.github/workflows/docker-publish.yml

```
@@ -0,0 +1,18 @@
+jobs:
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)